.swiftformat

```diff
@@ -1,5 +1,20 @@
 #Disable all and explicitly enable them to avoid unintentionally activating new rules in newer versions of SwiftFormat
---disable all
+#--disable all
+
+--rules \
```

Comment:
@eviltofu shucks, so many files and include rule changes. Can you help select just a few files to illustrate what's changed and force push, so I don't have go through 136 files?

Reply:
To be clear, I mean like, only 5 `.swift` files instead of ~136